app/_locales/en/messages.json

```diff
@@ -1684,23 +1684,23 @@
   "statusNotConnected": {
     "message": "Not connected"
   },
-  "step1HardwareWallet": {
-    "message": "1. Connect Hardware Wallet"
+  "step1LedgerWallet": {
+    "message": "Download Ledger app"
   },
-  "step1HardwareWalletMsg": {
-    "message": "Connect your hardware wallet directly to your computer."
+  "step1LedgerWalletMsg": {
+    "message": "Download, set up, and enter your password to unlock $1."
```

Comment:
Nit: In general we should include a `description` key for _all_ of these messages (we had agreed to commit to this as a team, but the linter hasn't been updated yet to reflect that), but we _especially_ should have descriptions whenever a substitution is used, to explain what will be substituted for `$1`.

The same goes for the other messages as well - I noticed a few others use substitutions.

Reply:
++